google-home-community.groovy

```diff
@@ -1960,6 +2254,58 @@ private executeCommand_ThermostatSetMode(deviceInfo, command) {
     ]
 }
 
+@SuppressWarnings('UnusedPrivateMethod')
+private executeCommand_TimerAdjust(deviceInfo, command) {
+    checkMfa(deviceInfo.deviceType, "TimerAdjust", command)
```

Comment:
All of these calls to `checkMfa` need to match an entry in the trait's `commands` list.  As is, this second arg in this call should just be "Adjust", but the commands should probably be more descriptive; i.e. "Adjust Timer", "Pause Timer", etc.  These are user-facing strings, so they should be plain English with normal spacing rather than CamelCase.

Same comment for `SoftwareUpdate` above.

Reply:
Thanks for clarifying this.  I wasn't 100% clear on how those two functions linked.  Corrected as well.